src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -465,7 +465,7 @@ def insertContainerRules(self):
                               grouping="ALL",
                               comment=ruleComment,
                               meta=self.metaData)
-            if not rseName.endswith("_Tape"):
+            if "Tape" not in rseName:
```

Comment:
This is a risky check, and given that we never produce data at a `Tape` endpoint, I would suggest to revert this change.

Reply:
Agreed, the way how this check has been implemented originally is better.